core/cache/model_test.go

```diff
@@ -135,6 +136,88 @@ func (s *ModelSuite) TestApplicationNotFoundError(c *gc.C) {
 	c.Assert(errors.IsNotFound(err), jc.IsTrue)
 }
 
+func (s *ModelSuite) TestMachineNotFoundError(c *gc.C) {
+	m := s.newModel(modelChange)
+	_, err := m.Machine("nope")
+	c.Assert(errors.IsNotFound(err), jc.IsTrue)
+}
+
+func (s *ControllerSuite) TestWatchMachineStops(c *gc.C) {
```

Comment:
These test work best with the ControllerSuite, but the piece tested isn't in controller.go... I debated about what file to place them in.  Or rewrite with new methods in export_test.go